capture.sh

```diff
@@ -11,9 +20,28 @@ else
   exit 1
 fi
 
+PERIODI=4 #default to 4 seconds between captures. Can be changed with --period command line option
+if [ ! -z "${period}" ]
+    then
+      PERIODI="$period"
+fi
```

Comment:
Do you think we need some kind of validation for this value?

Reply:
I added evaluation for both -app and --period. Good idea!